Docker/securityserver/Dockerfile

```diff
@@ -57,17 +58,16 @@ RUN --mount=type=bind,source=build/packages,target=/tmp/packages \
     # Install
     && pg_ctlcluster 16 main start \
     && apt-get update \
-    && apt-get -y -qq install xroad-securityserver xroad-autologin xroad-addon-opmonitoring \
+    && apt-get -y -qq install xroad-securityserver xroad-autologin xroad-addon-hwtokens xroad-addon-opmonitoring \
```

Comment:
Could `xroad-addon-hwtokens` be installed final-hwtoken stage instead? This could probably mean, that clean-up steps would need to be moved and duplicated to final stages too.